trench/models.py

```diff
@@ -74,6 +77,19 @@ class MFAMethod(Model):
     class Meta:
         verbose_name = _("MFA Method")
         verbose_name_plural = _("MFA Methods")
+        constraints = (
+            UniqueConstraint(
```

Comment:
👍

Actually I'm surprised that this constraint wasn't there since the beginning 😅